app/celery/service_callback_tasks.py

```diff
@@ -141,3 +141,21 @@ def process_update_with_notification_id(self, notification_id):
                 """Retry: send_delivery_status_to_service has retried the max num of times
                  for notification: {}""".format(notification_id)
             )
+
+
+def create_encrypted_callback_data(notification, service_callback_api):
+    from app import DATETIME_FORMAT, encryption
+    data = {
```

Comment:
Should this be moved into the notifications model - its basically serialising?

Reply:
Probably not, because the service_callback_api stuff is in there too.